web/tests/functional/diff_local/test_diff_local.py

```diff
@@ -77,15 +77,16 @@ def setup_class(self):
             'skip_list_file': skip_list_file,
             'check_env': test_env,
             'workspace': TEST_WORKSPACE,
-            'checkers': [],
+            'checkers': ['-d', 'clang-diagnostic'],
```

Comment:
Sorry, it seems like you have to beat this into me. Don't we disable all compiler warnings out of the box? Why is this needed? Why did you need to change almost all invocations of CodeChecker in these tests? Its a mess.

Reply:
Some `clang-diagnostic-...` checkers have been added to the "default" profile (see the changes of `clang-tidy.json`), that's why some of them appeared in tests files. So either I rewrite all test files to eliminate the source of the report, or I disable `clang-diagnostic-...` checkers in order to remain conform with the previous behavior.

Reply:
Aha, I see. In that case, can we not move the profile changed to its own PR?